qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_sampling_measurement.py

```diff
@@ -0,0 +1,98 @@
+import warnings
+
+import numpy
+
+from qcodes import ParameterWithSetpoints
+from qcodes.instrument_drivers.Keysight.keysightb1500 import MessageBuilder,\
+    constants
+from qcodes.instrument_drivers.Keysight.keysightb1500.KeysightB1500_module \
+    import parse_fmt_1_0_response, FMTResponse
+
+
+class MeasurementNotTaken(Exception):
+    pass
+
+
+class SamplingMeasurement(ParameterWithSetpoints):
+    """
+    Performs sampling measurement using semiconductor
+    parameter analyzer B1500A.
+    """
+
+    _timeout_response_factor = 10.00
+    # This factor is a bit higher than the ratio between
+    # the measured measurement-time and the calculated measurement
+    # (from the user input). Check :get_raw: method to find its usage.
+
+    def __init__(self, name, **kwargs):
+        super().__init__(name, **kwargs)
+        self.data = FMTResponse(None, None, None, None)
+
+    def get_raw(self):
+        """
+        Sets up the measurement by calling :_set_up:
```

Comment:
I don't think the docs should refer to private methods. Also I don't think `:something:` is valid markup.
I think this should read perform measurement ...

Reply:
Right. This docstring has been reworked.